src/azure-cli/azure/cli/command_modules/sqlvm/_util.py

```diff
@@ -3,6 +3,20 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure.core.exceptions import HttpResponseError
+from azure.core.paging import ItemPaged
+from azure.cli.command_modules.vm._client_factory import cf_log_analytics_data_sources
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.command_modules.vm.custom import (
+    list_extensions,
+    _get_log_analytics_client,
+    set_extension,
+    extension_mappings,
+    _WINDOWS_OMS_AGENT_EXT
+)
```

Comment:
Add VM owner @zhoxing-ms for review.

Do you agree that `sqlvm` binding with `vm` functions?

Reply:
I think it is OK to call the public method from `vm` module. 
For calling private methods, this way does not conform to the code specification, and it is easy to ignore the calling relationship. So I personally suggest that either implement `_get_log_analytics_client()` method in `sqlvm` module *(the code of the `_get_log_analytics_client()` method is very simple)* or change the `_get_log_analytics_client()` to a public method

Reply:
Makes sense. Copied all private variables to sqlvm as it was minimal code.